validators/types.go

```diff
@@ -34,6 +34,19 @@ type Response struct {
 	Timestamp   time.Time         `json:"timestamp"`
 }
 
+// Status is the message sent to the payload tracker
+type Status struct {
+	Service     string `json:"service"`
+	Source      string `json:"source,omitempty"`
+	Account     string `json:"account"`
+	RequestID   string `json:"request_id"`
+	InventoryID string `json:"inventory_id"`
+	SystemID    string `json:"system_id"`
+	Status      string `json:"status"`
+	StatusMsg   string `json:"status_msg"`
+	Date        string `json:"date"`
```

Comment:
You can use `time.Time` and forego the `Format` calls elsewhere.

Reply:
does time.Time give you an isoformat though? I think the payload-tracker is dependent on the time we send to be the time it expects

Reply:
The JsonMarshaller for `time.Time` does time.RFC3339 formatting for you.

https://golang.org/src/time/time.go?s=38016:38059#L1248

Reply:
word. I'll get that dropped in